advanced/approve_circleci_job/README.md

```diff
@@ -21,33 +21,35 @@ Once approved, Sym will automatically resume the workflow after the `wait_for_sy
 orbs:
   sym: sym/sym@1.0.0
```

Comment:
You'll want to bump this version number to the latest after we cut a new release with the context feature